sys/net/gnrc/netif/gnrc_netif_util.c

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright (C) 2018 Freie Universität Berlin
```

Comment:
Why a new file? IMHO this can just go into `gnrc_netif.c`

Reply:
I didn't want to clutter up `gnrc_netif.c` with auxiliary functions, but I'll move it there then.